README.md

```diff
@@ -58,42 +47,57 @@ Active Record to grab the article with the `id` that is in the `params` and set
+`new.erb` view.  This view will be a blank form that should submit a `POST` request
+to `/articles`.  (Look up the method and action attributes for HTML forms if you
+aren't sure how to do this).
 
-You'll need to make sure the edit form includes:
+Now you will need to tell your controller what to do when your form sends that
+`POST` request, so create a route on your controller `post '/articles'` that creates a new
+article from the `params` from the form, then redirects to that new article's show page.
 
+
+### Update
+
+The Update CRUD action corresponds to the edit controller action and view.
+
+Create a controller action, `get '/articles/:id/edit'`, that renders the view
+`edit.erb`. This view should contain a form to update a specific article--similar to the form
+you made for a new article, but the fields should be pre-populated with the existing title and
+content of the article. Although we want to send a `PATCH` request to `/articles/:id`, we have to
+be a little sneaky to trick HTML into letting us do something besides a `GET` or a `POST`.  
+Define the controller action `patch '/articles/:id'`, but your form will be configured to send
+a `POST` in the `method` attribute, and include:
 ```html
 <input id="hidden" type="hidden" name="_method" value="PATCH">
 ```
-
+ 
 **Reminder:** Remember to add the `use Rack::MethodOverride` to your
 `config.ru` file so that your app will know how to handle `PATCH`, `PUT`, and `DELETE`
 requests!
 
+(This is confusing and weird, and you should ask questions about it!)
+
 ### Delete
 
 The Delete CRUD action corresponds to the delete controller action, `delete
 '/articles/:id'`. To initiate this action, we'll add a "delete" button to the
-show page. This button will be in a form, but since the form isn't visible by
-default, you should only be able to see the button (intriguing, I know). The
-form will send a request to the delete controller action, where we will
-identify the article to delete and delete it.  Then, the action should redirect
+*show page* (i.e., deleting won't have its own view). This button will be in a form, 
+but the form won't have any other fields in it... just the single button. 
+The form will send a request to the delete controller action, 
+where we will identify the article to delete and delete it.  Then, the action should redirect
 to the index of all articles — we can't go back to the show page, since the
 article has been deleted!
 
 #### Making our Delete "Button"
 
-In order to make a form that looks like a button, all we need to do is make a
-form that has no input fields, only a "submit" button with a value of "delete".
-So, give your form tag a method of `POST` and an action of `"/articles/:id'`.
-Make sure to dynamically set the `:id` of the form action! You'll also need to
+Give your form tag a method of `POST` and an action of `"/articles/:id'`.
+Make sure to dynamically set the `:id` of the form action to reflect the! You'll also need to
```

Comment:
Fix for "form action to reflect the!":
...form action to reflect the id of the article!